modules/scripts/startup-script/variables.tf

```diff
@@ -268,9 +268,3 @@ variable "install_cloud_rdma_drivers" {
   type        = bool
   default     = false
 }
-
-variable "set_ofi_cloud_rdma_tunables" {
```

Comment:
Instead of deleting it has to be "deprecated"

Reply:
Added deprecated variable entry